newIDE/app/src/EventsSheet/SelectionHandler.js

```diff
@@ -14,6 +14,14 @@ export type InstructionContext = {|
   indexInList: number,
 |};
 
+// Used for history management.
+type EventContainer = {| eventContainer: gdBaseEvent |};
```

Comment:
Not sure about the naming here. There is a component `EventContainer` that is misleading. Maybe `parentEvent`?

Reply:
Or maybe just `event`

Reply:
Long live to the Cmd/Ctrl + D